lib/jsonapi_errors_handler/errors/forbidden.rb

```diff
@@ -8,7 +8,7 @@ class Forbidden < ::JsonapiErrorsHandler::Errors::StandardError
       def initialize(message: nil)
         super(
           title: 'Forbidden request',
-          status: 403,
+          status: '403',
```

Comment:
Apps using this gem can have their own custom errors being already initialized.
Instead of forcing the clients to remember about initializing the status using strings and break the backward compatibility, you could just do the update in the standard error by adding `to_s` in the status key. https://github.com/driggl/jsonapi_errors_handler/blob/master/lib/jsonapi_errors_handler/errors/standard_error.rb#L23

This would keep the backward compatibility.